tests/imagebasedupgrade/internal/ibuclusterinfo/ibuclusterinfo.go

```diff
@@ -25,9 +26,24 @@ func SaveClusterInfo(upgradeVar *imagebasedupgradeparams.ClusterStruct) error {
 		return err
 	}
 
+	csvList, err := olm.ListClusterServiceVersionInAllNamespaces(TargetSNOAPIClient)
+
+	if err != nil {
+		glog.V(100).Infof("Could not retrieve csv list")
+
+		return err
+	}
+
+	var installedCSV []string
+
+	for _, csv := range csvList {
+		installedCSV = append(installedCSV, csv.Object.Name)
```

Comment:
duplicated entries would end up here from operators that are installed in all the namespaces(or `ListClusterServiceVersionInAllNamespaces` provides already filtered list or it's OK to have duplicated entries?)

Reply:
Currently there are no cluster wide operators installed on these nodes but I updated it so that it adds an operator to the list only if it's not there.